q2_assembly/helpers/helpers.py

```diff
@@ -109,3 +114,24 @@ def collate_alignments(alignments: BAMDirFmt) -> BAMDirFmt:
             duplicate(_alignment, os.path.join(collated_alignments.path, filename))
 
     return collated_alignments
+
+
+def collate_genomes(
+    genomes_in: Union[DNAFASTAFormat, GenomeSequencesDirectoryFormat]
+) -> GenomeSequencesDirectoryFormat:
+    genomes_dir = GenomeSequencesDirectoryFormat()
+    if isinstance(genomes_in[0], DNAFASTAFormat):
+        for genome_file in genomes_in:
+            for genome in genome_file.view(DNAIterator):
+                name = genome.metadata["description"].split(",")[0]
```

Comment:
Two questions here:
1. why are there commas in some headers?
2. why are you using the description to get the name? didn't we discuss using the sequence ID as a new name?

Reply:
1. The commas are part of the description itself, e.g: >NZ_CP020468.1 Actinomyces gaoshouyii strain pika_114 chromosome, complete genome, this is from one of the files downloaded by QUAST.
2. I used the description to get the species name as they do it in QUAST, but maybe you are right, we decided on using the ID. I am correcting this now.